afs/media/js/views/components/workflows/analysis-areas-workflow/analysis-areas-annotation-step.js

```diff
@@ -386,7 +386,25 @@ define([
 
             self.savingTile(true);
             params.form.lockExternalStep('image-step', true);
-            self.savingMessage(`Saving Analysis Area ...`);
+
+            self.savingMessage(`This step is saving ...`);
+            const savingMessages = [
+                `Analysis Area Name and Classification`,
+                `Relationship between Analysis Area, Project and Parent Object (${self.physicalThingName()})`,
+                `It may take a while, so do not move away from this step.`
+            ];
+            let i = 0;
+            const showMessage = setInterval(() => {
+                if (i < savingMessages.length) {
+                    self.savingMessage(savingMessages[i++]);
+                } else {
+                    clearInterval(showMessage);
+                }}, '2000'
+            );
+
+            const showExtraMessage = setTimeout(() => {
+                self.savingMessage(`It takes longer than usual. Thank you for your patience.`);
```

Comment:
Let's change this to "This is taking longer than usual.  Thank you for your patience."